### PR TITLE
Kevincai/week 4 (Mainly pickup and delivery pages)

### PR DESCRIPTION
**Pull request by kevinjcai**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🛍 CC: @EthanAuyeung

### CODE REVIEW

File: src/app/delivery/page.tsx [hunk 1]
```diff
@@ -62,21 +64,32 @@ export default function App() {
...
-            <Heading1 style={{ marginBottom: '38px' }}>Shipping</Heading1>
-            <Normal700Text>Name</Normal700Text>
-            <InformationText>
+          <ForceColumnDiv>
+            <BackButtonDiv>
+              <BackButton destination="/storefront" />
+            </BackButtonDiv>
+
+            <Heading4Bold style={{ marginLeft: '41px', marginTop: '20px' }}>
+              Delivery
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/page.tsx]: Shipping

File: src/app/delivery/page.tsx [hunk 2]
```diff
@@ -62,21 +64,32 @@ export default function App() {
...
-          <InformationContainer>
-            <Heading1 style={{ marginBottom: '38px' }}>Shipping</Heading1>
-            <Normal700Text>Name</Normal700Text>
-            <InformationText>
+          <ForceColumnDiv>
+            <BackButtonDiv>
+              <BackButton destination="/storefront" />
+            </BackButtonDiv>
+
+            <Heading4Bold style={{ marginLeft: '41px', marginTop: '20px' }}>
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/page.tsx]: suppose to be heading1

Comment (BuyankhuuTsCAl) [reply src/app/delivery/page.tsx]: can you fix the top margin?

File: src/app/delivery/styles.ts [hunk 3]
```diff
@@ -155,3 +132,65 @@ export const OutterDiv = styled.div`
...
+
+  /* Body 1 - Text */
+  font-family: 'Public Sans';
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+`;
+
+export const Heading4Bold = styled.p`
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/styles.ts]: this exists within another file

File: src/app/delivery/styles.ts [hunk 4]
```diff
@@ -155,3 +132,65 @@ export const OutterDiv = styled.div`
...
+  width: calc(100% - 40px); /* Subtracting padding */
+  height: 50px;
+  border-radius: 4px;
+  padding-left: 20px;
+  font-size: 20px;
+  color: var(--Black, #101010);
+
+  /* Body 1 - Text */
+  font-family: 'Public Sans';
+  font-size: 20px;
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/styles.ts:178]: this exists within the text file make sure ot use that

File: src/app/delivery/styles.ts [hunk 5]
```diff
@@ -155,3 +132,65 @@ export const OutterDiv = styled.div`
...
+  margin-right: 20px; /* Space between label and input */
+`;
+
+export const Input = styled.div`
+  width: calc(100% - 40px); /* Subtracting padding */
+  height: 50px;
+  border-radius: 4px;
+  padding-left: 20px;
+  font-size: 20px;
+  color: var(--Black, #101010);
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/styles.ts]: use the color file

File: src/app/delivery/styles.ts [hunk 6]
```diff
@@ -155,3 +132,65 @@ export const OutterDiv = styled.div`
...
+export const InformationField = styled.div`
+  width: 100%;
+  margin-top: 14px;
+  margin-bottom: 14px;
+  display: flex;
+  align-items: center;
+`;
+
+export const Label = styled.div`
+  color: var(--Black, #101010);
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/styles.ts]: color + text file

File: src/app/delivery/styles.ts [hunk 7]
```diff
@@ -155,3 +132,65 @@ export const OutterDiv = styled.div`
...
+
+export const InformationContainer = styled.div`
+  width: 629px;
+  height: 327px;
+  margin: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: left;
+  background: ${COLORS.white}; /* Assuming white is defined in your COLORS */
```
Comment (BuyankhuuTsCAl) [inline src/app/delivery/styles.ts:144]: example rn
